lib/ExtUtils/MM_Any.pm

```diff
@@ -1671,7 +1687,7 @@ sub _mymeta_from_meta {
     # rolled their own tarball rather than using "make dist".
     if ($meta->{generated_by} &&
         $meta->{generated_by} =~ /ExtUtils::MakeMaker version ([\d\._]+)/) {
-        my $eummv = do { no warnings; $1+0; };
+        my $eummv = do { no warnings; eval "$1" + 0; }; # deal with underbars gracefully
```

Comment:
This equals `eval("$1" + 0)`, I suspect you meant `eval("$1") + 0`

Reply:
The better way to do this is `tr/_//d`

Reply:
Or let `version` take care of it. Or not care at all about the part behind the underscore.

Reply:
> I suspect you meant eval("$1") + 0

Yes. I pushed a fix. I dont see the point in rewrting this to use tr///. Eval is fine IMO.